tests/unit/unit1604.c

```diff
@@ -304,8 +304,9 @@ UNITTEST_START
     char *flagstr = NULL;
     char *received_ccstr = NULL;
     char *expected_ccstr = NULL;
+    SANITIZEcode res;
 
-    CURLcode res = sanitize_file_name(&output, data[i].input, data[i].flags);
+    res = sanitize_file_name(&output, data[i].input, data[i].flags);
```

Comment:
You could also just merge these two lines into a single statement...

Reply:
I could, but then it would exceed the line length limit.